modular_zubbers/code/datums/quirks/neutral_quirks/pseudo_respiration.dm

```diff
@@ -0,0 +1,28 @@
+/datum/quirk/pseudo_respiration
+	name = "Pseudo-Respiration"
+	desc = "(Hemophage only) Despite your condition you for whatever reason still require to breathe."
+	icon = FA_ICON_MASK_VENTILATOR
+	value = 0
```

Comment:
quirk value is by default 0, but i would honestly make it negative as it removes a beneficial part